### PR TITLE
JIRA-003: Restrict money related programs to owner only

### PR DESCRIPTION
**Pull request by superseal**
description sir...............................

### CODE REVIEW

**Comment by juanfelipepc** [comment]
Please add description kind sir.......................................................... also add uhhh jira

File: raisin/commands.py [hunk 1]
```diff
@@ -60,7 +53,7 @@ def cringe_words(message):
 
     # Random choice
     elif command == 'pick':
-        if len(args) > 1:
+        if len(args) >= 2:
```
Comment (juanfelipepc) [inline raisin/commands.py:56]: what is habbeninges heres :DDDD

Comment (superseal) [reply raisin/commands.py:56]: fug :DDDDD

**Review by juanfelipepc — APPROVED** [review]
Approved cause reasons

**Review by juanfelipepc — APPROVED** [review]
sos groso

**Pull request merged**